### ISSUE TITLE
Add select/unselect all function to imports & list

### ISSUE THREAD

**Issue by zephyrr** [adiwg/mdEditor#143]
### Issue Description:

There is a very useful checkbox at the top left for selecting or unselecting all for each of the metadata records and contact lists on the "Export" page (https://jlblcc.github.io/mdEditor/#/export)

Please add similar functionality to "Import" (https://jlblcc.github.io/mdEditor/#/import).  I have an mdEditor file with 150+ contacts (all contacts from all projects), and it would be handy to be able to easily select a few contacts to import (to supplement the current metadata record being edited).

Such functionality would also be useful on the lists (https://jlblcc.github.io/mdEditor/#/import and https://jlblcc.github.io/mdEditor/#/contacts) if you implement any group actions, like delete.

**Comment by jlblcc** [comment]
Yep, this has been the plan all along. Now we have an issue to track it. :wink:

**Comment by jlblcc** [comment]
Fixed by 1688d4e96d9c779e31443b076d252b6832ba3e17
